### ISSUE TITLE
Print (to PDF) function

### ISSUE THREAD

**Issue by fiffty-50** [fiffty-50/openPilotLog#93]
Required functionality for v 1.0

The database needs to be printable.

The solution should be built with customization in mind:
- what columns of the table are to be printed
- layouts and paper sizes
- print to different file types

However, the first implementation should follow the [EASA specifications](https://www.easa.europa.eu/sites/default/files/dfu/AMC%20and%20GM%20to%20Part-FCL.pdf) for logbooks and print to PDF.

**Comment by roema** [comment]
Hi @fiffty-50 !

I will have a look at the PDF export. https://capzlog.aero/ has a pdf export which is exactly the same as the flight log (printed form). This way you can compare the entries in the digital logbook with the handwritten logbook.

Kind regards, Manu

**Comment by fiffty-50** [comment]
Fantastic, that's a great idea!

**Comment by roema** [comment]
Hi @fiffty-50!

I have created an pdf export example of my logbook. 

 
![FlightlogExport](https://github.com/fiffty-50/openPilotLog/assets/9306994/e517d5ad-ab26-4009-9bcf-f6f19ae66d3e)

**Comment by fiffty-50** [comment]
Hi @roema Looks great!

The only thing missing is the columns for logging FSTD (Simulator) Time, see p, 21 of the [specification](https://www.easa.europa.eu/sites/default/files/dfu/AMC%20and%20GM%20to%20Part-FCL.pdf).

How did you implement this?

**Comment by roema** [comment]
Hi @fiffty-50 !

Thanks!
Its an exact copy from my Logbook in Switzerland and column 11 (FSTD) is missing...

I will insert this column the next days.

I use this very simple pdf generation lib written in c: https://github.com/AndreRenaud/PDFGen

**Comment by fiffty-50** [comment]
I like it, looks like it wouldn't bee too complex to integrate!

**Comment by roema** [comment]
Yes! I think on weekend i am ready to create a pull request for the pdf export.
